dolly-backend-app/src/main/java/no/nav/dolly/service/NavigasjonService.java

```diff
@@ -35,11 +38,16 @@ public RsWhereAmI navigerTilIdent(String ident) {
 
         List<Testident> identsFound = identRepository.findByIdentIn(identer);
         if (!identsFound.isEmpty()) {
+            Page<Testident> bestillingerFromGruppePaginert = identService.getBestillingerFromGruppePaginert(identsFound.get(0).getTestgruppe().getId(), 0, 5000);
```

Comment:
5000 er for lav til å støtte de største gruppene, ser at 10070 finnes i dag ... 😄 
Fant/laget et query vha google 😃  som jeg tror vil løse det direkte:
```
select *
from (
         select ti.ident, count(*) over (order by bp.id desc )
         from test_ident ti, bestilling_progress bp
         where ti.ident = bp.ident
         and ti.tilhoerer_gruppe = <gruppeId>
         and bp.id = (select max(bps.id) from bestilling_progress bps where bps.ident = ti.ident)
     ) x where ident = '<ident>';
```
Da får du direkte indeks til posten du ønsker.

Reply:
Det var såpass ja, stemmer det 😂 
Åja supert, så smart! Da skal jeg teste det, takk 🤩